test/test_layout_base.rb

```
@@ -329,3 +329,178 @@ def test_failed_to_running_blocked
+        [
+          "task.name:{{ task.name }}",
+          "task.state:{{ task.state }}",
+          "task.duration:{{ task.duration }}",
+          "task.error_message:{{ task.error_message }}",
+          "execution.state:{{ execution.state }}",
+          "execution.pending_count:{{ execution.pending_count }}",
+          "execution.done_count:{{ execution.done_count }}",
+          "execution.completed_count:{{ execution.completed_count }}",
+          "execution.failed_count:{{ execution.failed_count }}",
+          "execution.total_count:{{ execution.total_count }}",
+          "execution.root_task_name:{{ execution.root_task_name }}"
```

RuboCop (1.84.0)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)